app.js

```diff
@@ -1,9 +1,26 @@
 import express from "express";
+import cors from "cors";
+
+import { AuthorizationController } from "./controllers/authorization.controller.js";
+
+import { checkToken } from "./middleware/token.middleware.js";
```

Comment:
checkToken is not so clear name. This part is responsible for authorization checks, and if we will move our API to the basic auth, for example, postfix "Token" will be not valid, so I would suggest using more generic names for such a things, like "ensureAuthorized"